app/graphql/resolvers/article_query_function.rb

```diff
@@ -0,0 +1,12 @@
+class Resolvers::ArticleQueryFunction < GraphQL::Function
+
+    def select_published(articles)
+        # nil articles will be filtered out
+        if articles.nil?

```

Comment:
`return nil if articles.nil?` is more ruby-esque

Reply:
Thanks for the suggestion, I changed it.